README.md

```diff
@@ -85,9 +79,9 @@ go get -u -v github.com/btcsuite/btcwallet/...
 
 ## Getting Started
 
-The following instructions detail how to get started with btcwallet
-connecting to a localhost btcd.  Commands should be run in `cmd.exe`
-or PowerShell on Windows, or any terminal emulator on *nix.
+The following instructions detail how to get started with btcwallet connecting
+to a localhost testnet btcd.  Commands should be run in `cmd.exe` or PowerShell
```

Comment:
Actually, I'm ok with that.  If you're a new user and using it directly on the command line instead of through another client, I'd expect you to have some knowledge about wallet security.